docs/pages/kicon.vue

```diff
@@ -3,6 +3,8 @@
   <DocsPageTemplate apiDocs>
     <DocsPageSection title="Overview" anchor="#overview">
       See <DocsInternalLink href="icons" text="the page on icons" /> for design guidance and a list of available icons.
+      If an invalid icon is used, in development Vue.js validation will warn about the error.
+      The icon will display as a broken image <KIcon icon="brokenImage" /> icon.
```

Comment:
```suggestion
      See <DocsInternalLink href="icons" text="the page on icons" /> for design guidance and a list of available icons. If an invalid icon is used, in development Vue.js validation will warn about the error. The icon will display as a broken image <KIcon icon="brokenImage" /> icon.
```

Reply:
FYI, it's generally easier if content (particularly in `docs/pages`) is soft-wrapped by editors rather than hard-wrapped in the file itself:

* easier editing experience because writers don't need to worry about endline placement
* better diffs in git history over time

Kolibri Tools max-length rules are overwritten here:

https://github.com/learningequality/kolibri-design-system/blob/d24d3d590c6fe82ca8c475c722511ed9410a1da4/.eslintrc.js#L13-L15

However, it looks like I never actually wrote this guidance anywhere visible. Sorry!